torchaudio/functional/functional.py

```diff
@@ -479,6 +480,127 @@ def _mel_to_hz(mels: Tensor, mel_scale: str = "htk") -> Tensor:
     return freqs
 
 
+def _hz_to_bark(freq: float, bark_scale: str = "traunmuller") -> float:
+    r"""Convert Hz to Barks.
+
+    Args:
+        freqs (float): Frequencies in Hz
```

Comment:
`freqs` here is inconsistent to the `freq` variable used in the code/function call

Reply:
Done; I decided to change the variable to make it consistent with the plural meaning and with the definition in `_bark_to_hz`. By the way, I noticed the same inconsistency in `_hz_to_mel`. It could be fixed without much analysis because those functions are called only by `melscale_fbanks`.